pkg/clusterclientstore/queue.go

```diff
@@ -25,6 +26,7 @@ func (s *Store) secretWorker() {
 
 func (s *Store) bindEventHandlers() {
 	enqueueSecret := func(obj interface{}) { enqueueWorkItem(s.secretWorkqueue, obj) }
+	enqueueSecretAfterDuration := func(obj interface{}) { s.enqueueWorkItemAfterDuration(s.secretWorkqueue, obj) }
```

Comment:
I believe there is no dedicated closure around the arguments of this function, therefore it might be simpler to call `s.enqueueWorkItemAfterDuration` with no extra syntactic sugar function wrapper: it's redefined in a few blocks below every time and is being called only once per block.

Reply:
It is being redefined but with a different workqueue. I can make more readable and unwrap it...